pkg/terraform/terraform.go

```diff
@@ -127,3 +134,30 @@ func unpackAndInit(dir string, platform string) (err error) {
 	}
 	return nil
 }
+
+func setupEmbeddedPlugins(dir string) error {
+	execPath, err := os.Executable()
+	if err != nil {
+		return errors.Wrapf(err, "failed to find path for binary %q", "openshift-install")
+	}
+
+	pdir := filepath.Join(dir, "plugins")
+	if err := os.MkdirAll(pdir, 0777); err != nil {
+		return err
+	}
+	for name := range plugins.KnownPlugins {
+		dst := filepath.Join(pdir, name)
+		if runtime.GOOS == "windows" {
+			dst = fmt.Sprintf("%s.exe", dst)
+		}
+		if _, err := os.Stat(dst); err == nil {
+			// stat succeeded, the plugin already exists.
+			continue
+		}
+		logrus.Debugf("Symlinking plugin %s src: %q dst: %q", name, execPath, dst)
```

Comment:
NIT: Can we move this to trace?

Reply:
I'm fine witg debug or trace, but if we go with trace we should list it [here][1].

[1]: https://github.com/openshift/installer/blob/4d230b6d047509a57ab8095ed801a047b203c0e3/cmd/openshift-install/main.go#L47